specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTranscriptJobsTaskSubmitRequest.json

```diff
@@ -0,0 +1,77 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2022-04-01-preview",
+    "Endpoint": "{Endpoint}",
+    "jobId": "{Job ID}",
+    "body": {
+      "displayName": "Redacting PII data from transcribed audio",
+      "analysisInput": {
+        "conversations": [
+          {
+            "id": "conv-01",
+            "language": "en",
+            "modality": "transcript",
+            "domain": "neutral",
+            "conversationItems": [
+              {
+                "participantId": "1",
+                "id": "turn-01",
+                "modality": "transcript",
+                "language": null,
```

Comment:
Customers should never submit `null` values (even if technically they can). Either omit this parameter completely, or specify the language.

Reply:
sure. This should not be present.